upload/install/opencart.sql

```diff
@@ -3436,6 +3443,7 @@ INSERT INTO `oc_setting` (`store_id`, `code`, `key`, `value`, `serialized`) VALU
 (0, 'config', 'config_mail_alert', '["order"]', 1),
 (0, 'config', 'config_captcha', 'basic', 0),
 (0, 'config', 'config_captcha_page', '["review","return","contact"]', 1),
+(0, 'ecb', 'ecb_status', '1', 0),
```

Comment:
please add currency prefix

Reply:
My mistake. Also I forgot to add permissions :)